src/components/side_panel/conditional_formatting/cf_editor/cf_editor.ts

```diff
@@ -280,6 +272,13 @@ export class ConditionalFormattingEditor extends Component<Props, SpreadsheetChi
     }
   }
 
+  onSave() {
```

Comment:
Change something, let's say a color (which dispatches), then click on save: the exact same command is dispatched (and accepted). We could add an `allowDispatch` to rejects `ADD_CONDITIONAL_FORMAT` commands that wouldn't change anything to the CF